### PR TITLE
CGMES updates: reduce size of changelog not recording state variables

### PR DESCRIPTION
**Pull request by zamarrenolm**
Signed-off-by: Luma <zamarrenolm@aia.es>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
The changelog that records changes in IIDM attributes is used to perform updates in the CGMES triplestore when the user requests a CGMES export.
State variables (voltages, angles, flows at terminals) are not recorded explicitly in the changelog, as its values will be systematically obtained during the export to build the CGMES SV profile from scratch. No selective update of these attributes is required.

**What is the current behavior?** *(You can also link to an open issue here)*
These attributes were previously stored in the changelog, and later ignored during the update.

**What is the new behavior (if this is a feature change)?**
Changelog size is reduced, and less operations are perform during update.

**Other information**:
The tests for CGMES update have been modified to remove size limitations on the number of changes performed.

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/ShuntCompensatorToShuntCompensator.java [hunk 1]
```diff
@@ -15,7 +15,6 @@
 public class ShuntCompensatorToShuntCompensator extends IidmToCgmes {
 
     ShuntCompensatorToShuntCompensator() {
-        ignore("q");
 
         simpleUpdate("bPerSection", "cim:LinearShuntCompensator.bPerSection", CgmesSubset.EQUIPMENT);
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/ShuntCompensatorToShuntCompensator.java:19]: Now, we have non linear shunt compensator, so it means that we also have to support that for the CGMES update. I wonder if the code is working for that kind of equipment.

Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/ShuntCompensatorToShuntCompensator.java:19]: I will have to review. Current export does not support `NonLinearShuntCompensators`.

Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/ShuntCompensatorToShuntCompensator.java:19]: I propose to add export support for that equipment in a separate, specific PR.

Comment (annetill) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/ShuntCompensatorToShuntCompensator.java:19]: Yes, great. I am going to open an issue to keep that in mind and I am going to merge this PR as it is.

**Pull request merged**